requirements-dev.txt

```diff
@@ -24,3 +24,7 @@ releases==1.5.0
 
 # Release
 bumpversion==0.5.3
+
+# eth-tester
+git+https://github.com/pcppcp/eth-tester.git@add_status#egg=eth-tester
+py-evm==0.2.0a18
```

Comment:
when using eth-tester[py-evm] this shouldn't be required.

Reply:
evm 0.2.0a26 wasn't compatible with py-evm, but apparently they've fixed it in the new release... I'll update the requirements.